src/net.cpp

```diff
@@ -453,10 +453,9 @@ CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCo
         }
     }
 
-    /// debug print
     LogPrint(BCLog::NET, "trying connection %s lastseen=%.1fhrs\n",
-        pszDest ? pszDest : addrConnect.ToString(),
-        pszDest ? 0.0 : (double)(GetAdjustedTime() - addrConnect.nTime)/3600.0);
+             pszDest ? pszDest : addrConnect.ToString(),
+             CountHoursDouble(pszDest ? 0h : AdjustedTime() - addrConnect.nTime));
```

Comment:
Maybe something that can be done after C++20 with https://en.cppreference.com/w/cpp/chrono/duration/operator_ltlt ? Not sure if worth it right now

Reply:
On a second thought it might probably be best to introduce a chrono auto-formatter to pick an appropriate precision and type at run time depending on the value of the duration.